bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -459,6 +459,17 @@ public static BXml createXmlValue(BXmlQName startTagName, BString defaultNsUriVa
         return XmlFactory.createXMLElement(startTagName, defaultNsUriVal, readonly);
     }
 
+    /**
+     * Create attribute map with an XML.
```

Comment:
Is this doc comment correct?

Reply:
Did you mean something like `Create attribute map with a qualified name string and namespace prefix.`?

Reply:
No, why would this create a map? We are just creating an xml qualified name right.

Reply:
Am I missing something here?

Reply:
I took the doc comment from existing overloaded methods of `createXmlQName`. That's why I didn't change this method topic. I will update this. Anyway, do I have to change the others too (e.g. L474) ?

Reply:
Yes.
Yes please.